shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -709,6 +709,7 @@ \subsection{Block Header Transition}
       }
       \\~\\
       (\wcard,~\wcard,~\var{b_{prev}},~\wcard,~\var{es},~\var{ru},~\wcard,~\wcard)\leteq\var{nes}
+      % TODO: is es = (_ _ _ pp) ?
```

Comment:
yes, the protocol parameters are the fourth component of the epoch state. but I do not see what this is needed, we do not use the `pp` in the rule.

Reply:
we extract the max header size and max block size from the paramters